Manifest.toml

```diff
@@ -0,0 +1,299 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
This file should not be included in the commit. Probably the reason for the error.